### PR TITLE
added custom_settings.py

### PR DESCRIPTION
**Pull request by kurtisolo**
useful for having a single source of truth

### Contributor checklist

- [X] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

This change is intended to make it easier to modify specific settings without having to search for them throughout the entire project. It's always useful to have a single source of truth in such large projects.

### Related issue

- #683

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks for the PR on this, @kurtisolo! Quick note: no stress on the frontend formatting checks. I'm realizing that one file needed to get formatted after the changes I made in #665. Do you want to look into the backend formatting, or should I take care of that? :)

**Comment by kurtisolo** [comment]
@andrewtavis let me try to fix it =)

**Comment by kurtisolo** [comment]
@andrewtavis two different solutions:

- using `from backend.custom_settings import *  # noqa: F403` to make ruff ignore this line

- directly importing `custom_settings.py` along the project instead of settings.py. For example instead of:
```
from django.conf import settings
from rest_framework import pagination

class CustomPagination(pagination.PageNumberPagination):
    page_size = settings.PAGINATION_PAGE_SIZE
    page_size_query_param = "pageSize"
    max_page_size = settings.PAGINATION_MAX_PAGE_SIZE
```
then we should use:

```
from . import custom_settings
from rest_framework import pagination

class CustomPagination(pagination.PageNumberPagination):
    page_size = custom_settings.PAGINATION_PAGE_SIZE
    page_size_query_param = "pageSize"
    max_page_size = custom_settings.PAGINATION_MAX_PAGE_SIZE
```

I think both solutions are ok, we just have to add a couple line for documentation. I'm also working on the Wiki for backend documentation, so everything will be documented there

**Comment by andrewtavis** [comment]
I think it's ok to go the second route, @kurtisolo :) Makes it a bit more clear what the source of all this is. Thanks for the suggestions, and further for the work on the wiki! Really exciting to see what you're working on 😊

**Comment by andrewtavis** [comment]
I'm thinking as well that maybe we could look into a typical documentation website for this such that it's updated from the code itself? If you thought that working out something from [Sphinx](https://www.sphinx-doc.org/en/master/) could help, then let me know. Maybe we could use that as a baseline and then also include component-wise frontend documentation from #427 🤔

**Comment by kurtisolo** [comment]
@andrewtavis I've never used Sphinx, but it would definitely be a great opportunity to learn =) It would surely provide a solid and scalable foundation, so as to have comprehensive and high-quality documentation in the future.

**Comment by andrewtavis** [comment]
I'd be happy to set up a baseline instance if that'd be of interest, @kurtisolo. We could deploy to [readthedocs](https://about.readthedocs.com/?ref=readthedocs.com) at first and then shift the hosting later on when we have a bit more capacity?

**Comment by andrewtavis** [comment]
Heyo @kurtisolo: let me know what you think of [mkdocs-material](https://github.com/squidfunk/mkdocs-material) that's built on [mkdocs](https://www.mkdocs.org/) as a documentation solution. Not a Wiki, but serves the main purpose of being directly in the codebase. There's also Python autodoc functionality already from [mkdocstrings](https://mkdocstrings.github.io/usage/), with them also considering adding in TS support. The frontend docs would be mostly handled by the design system anyway, with us just needing to host and point people to the Histoire instance that has the examples.

File: backend/backend/custom_settings.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'''
+This file contains granular settings specifically related to the Activists project.
```
Comment (andrewtavis) [inline backend/backend/custom_settings.py:2]: Quick note on this, @kurtisolo: we always do `activist` in lower case :) No stress though! I can fix 😊

**Review by andrewtavis — APPROVED** [review]
Really great, @kurtisolo! Appreciate the attention to detail as always 😊

**Pull request merged**